lang/python/python3-readline/Makefile

```
@@ -0,0 +1,31 @@
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=python3-readline
+PKG_VERSION:=8.0.0
+PKG_RELEASE:=$(AUTORELEASE)
+
+PYPI_NAME:=gnureadline
+PKG_HASH:=61eef72ed02dad415ede49752e972a1d2bd8c35c1e4464565d7effd806c99476
+
+PKG_MAINTAINER:=Oskari Rauta <oskari.rauta@gmail.com>
+PKG_LICENSE:=GPL-1.0-only
+
+include ../pypi.mk
+include $(INCLUDE_DIR)/package.mk
+include ../python3-package.mk
+
+define Package/python3-readline
+	SECTION:=lang
+	CATEGORY:=Languages
+	SUBMENU:=Python
+	TITLE:=Python3 readline
+	DEPENDS:=+python3 +libreadline +libncurses
+endef
+
+define Package/python3-readline/description
+  Python readline support
```

```suggestion
  The standard Python readline extension statically linked against the GNU readline library.
```

[Copy&pasted from pypi.org](https://pypi.org/project/gnureadline/)

---

It is NOT statically linked. If it would, it would not depend on libreadline and libhistory...
Also, I found no mention about GPL-3.0, files did not contain any license information at all and pypi page only says GPL, so then I would say it's GPL-1.0 - but fine. Review you description recommendation.

---

GPL-3.0: is stated here:
- https://pypi.org/project/gnureadline/, but yeah, in description there can be anything.
Well, the LICENSE file is inside their GitHub repository: https://github.com/ludwigschwardt/python-gnureadline/blob/master/LICENSE

---

So, description in [their setup.py](https://github.com/ludwigschwardt/python-gnureadline/blob/master/setup.py#L20) is wrong? :thinking:

Well, your description for your package is really short and it should be extented to be more clear for others than you and me.

---

Ok. Did not look from there. Description should not lie. One can actually grab any software available on network and make it something else by patching the code completely. If original application was a patching software but now works as a wiping software, description should tell what it does, whether original description is what it is.

---

It is not clear if it's untrue. It's misleading. setup.py might say what ever; if I patch setup.py and make it dynamically linked- setup.py's description no longer is correct, but one I provided here is.

Make it:
The standard Python readline extension linked against the GNU readline library.

Then I'm good with it.

---

Suggest it to developers of gnureadline. ;-) I'm fine with your proposal.

---

I didn't patch it there.. Only here.. Patched software is forked software if patch comes from anyone else except original author. Therefor, as I patched it, I forked it, and when I forked it, I also gave it a new description 😉

---

Then it is a question, why do you need the patch in the first place. Sorry, but this is missing in PR description or git description.

---

Yes, we all just patch things for fun.. Compile it without and see yourself. Then try again, but this time- with patch. Reason for patch: all the pain goes away.